one/src/query.rs

```diff
@@ -0,0 +1,56 @@
+use clap::Args;
+use datafusion_cli::{exec::exec_from_repl, print_options::PrintOptions};
+use object_store::aws::AmazonS3Builder;
+
+#[derive(Args, Debug)]
+pub struct QueryOpts {
+    /// Endpoint of a Flight SQL server for the conclusion feed.
+    #[arg(
+        short,
+        long,
+        default_value = "http://127.0.0.1:5102",
+        env = "CERAMIC_ONE_FLIGHT_SQL_ENDPOINT"
+    )]
+    flight_sql_endpoint: String,
```

Comment:
I feel like the fact that it uses Flight as the API format is an implementation detail, the important thing is that this is a C1 node at the other end of this endpoint right?  Should this be called ceramic_one_endpoint or something like that?

Reply:
Yes, however ceramic-one exposes multiple endpoints(p2p, REST, flight sql). Maybe we call it `ceramic_one_query_endpoint`? As its purpose is to expose query access to the data?

Reply:
yep I like that

Reply:
Using `ceramic_one_query_endpoint` meant the env var was `ceramic_one_ceramic_one_query_endpoint` which is horrible. So I went with just `query_endpoint` as the `ceramic_one` context is already present